crates/parser/src/config/mod.rs

```diff
@@ -430,7 +586,7 @@ impl ParseConfig {
         let mut settings = schemars::gen::SchemaSettings::draft07();
         settings.option_add_null_type = false;
         settings.inline_subschemas = true;
-        settings.visitors.push(Box::new(RemoveDefaultNull {}));
+        //settings.visitors.push(Box::new(RemoveDefaultNull {}));
```

Comment:
I'll try to recount the backstory here. This was added originally because any field like:

```
#[serde(default)]
my_field: Option<String>
```

would result in a schema like:

```
"type": "string",
"default": null
```

Technically, the default doesn't validate against the schema. JsonForms also doesn't seem to handle `"type": ["string", "null"]` at all. So this was a hack to get those nulls removed. It's no longer necessary because `DefaultNullIsAutomatic` results in schemas that don't specify `type`, but rather use `const` values where the "Auto" value happens to be `null`.

I removed the commented out line and the `RemoveDefaultNull` type.